### ISSUE TITLE
[a11y]: The chat avatar should be aria hidden.

### ISSUE THREAD

**Issue by TazmanianDI** [carbon-design-system/carbon-ai-chat#92]
### Description

The chat avatar is an image without any alt-text or aria-label. Since there is visually hidden text just after the avatar that says `<chat name> said`, the image doesn't serve any purpose to a screen reader and so it should just be aria-hidden.

![Image](https://github.com/user-attachments/assets/8c6b7021-6587-4caa-97d4-66270e0315eb)

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-ai-chat/issues) for duplicate problems
